idunn/api/search.py

```diff
@@ -26,6 +27,10 @@ async def search(
     query.q = normalize(query.q)
     response = await get_autocomplete(query, extra)
```

Comment:
This shouldn't append often but it is a quite inefficient that we still call the geocoder when `q == ""`. You still check if `response.features` is empty, but I don't think there is any case where we expect the geocoder to give a result?